### PR TITLE
chore: fix some typos in comment

### PR DESCRIPTION
**Pull request by NinaLua**
## Description

 fix some typos in comment

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: transports/websocket/src/framed.rs [hunk 1]
```diff
@@ -511,7 +511,7 @@ fn parse_ws_dial_addr<T>(addr: Multiaddr) -> Result<WsAddress, Error<T>> {
             Some(Protocol::Ws(path)) => break (false, path.into_owned()),
             Some(Protocol::Wss(path)) => {
                 if dns_name.is_none() {
-                    tracing::debug!(addrress=%addr, "Missing DNS name in WSS address");
+                    tracing::debug!(addrres=%addr, "Missing DNS name in WSS address");
```
Comment (dariusc93) [inline transports/websocket/src/framed.rs]:
```suggestion
                    tracing::debug!(address=%addr, "Missing DNS name in WSS address");
```

Comment (NinaLua) [reply transports/websocket/src/framed.rs]: Awkwardly, I made even more outrageous changes..

Please review again.

**Review by jxs — DISMISSED** [review]
Thanks!

**Review by dariusc93 — APPROVED** [review]

**Review by jxs — DISMISSED** [review]

**Comment by drHuangMHT** [comment]
Ping @jxs for stalled PR

**Review by jxs — APPROVED** [review]

**Pull request merged**